lib/models/wallet_connect/request_event.dart

```diff
@@ -4,3 +4,8 @@ class RequestEvent {
   final int requestId;
   RequestEvent({required this.topic, required this.requestId});
 }
+
+const String wcRequestParamFrom = "from";
```

Comment:
It should be "fromID" and "toID" to work with the current dApp schema.

Reply:
dapp updated in this pr https://github.com/qubic/wallet-app-dapp/pull/7